<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMultiCopyForm</class>
 <widget class="QWidget" name="TMultiCopyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiCopy</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MultiCopy.qrc">
    <normaloff>:/MainIcon_32x32</normaloff>:/MainIcon_32x32</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QGroupBox" name="SrcBox">
      <property name="title">
       <string>Source(s)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QListWidget" name="SrcList"/>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="SrcAddFile">
           <property name="text">
            <string>&amp;Add File(s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SrcAddFolder">
           <property name="text">
            <string>Add &amp;Folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SrcRemove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SrcClear">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="SrcUp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SrcDown">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="DestBox">
      <property name="title">
       <string>Destination(s)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QListWidget" name="DestList"/>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="DestAddFolder">
           <property name="text">
            <string>A&amp;dd Folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DestRemove">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Re&amp;move</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DestClear">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>C&amp;lear</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Start">
       <property name="text">
        <string>&amp;Start</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="About">
       <property name="text">
        <string>&amp;About</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Settings">
       <property name="text">
        <string>S&amp;ettings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>SrcList</tabstop>
  <tabstop>SrcAddFile</tabstop>
  <tabstop>SrcAddFolder</tabstop>
  <tabstop>SrcRemove</tabstop>
  <tabstop>SrcClear</tabstop>
  <tabstop>SrcUp</tabstop>
  <tabstop>SrcDown</tabstop>
  <tabstop>DestList</tabstop>
  <tabstop>DestAddFolder</tabstop>
  <tabstop>DestRemove</tabstop>
  <tabstop>DestClear</tabstop>
  <tabstop>Start</tabstop>
  <tabstop>About</tabstop>
  <tabstop>Settings</tabstop>
 </tabstops>
 <resources>
  <include location="MultiCopy.qrc"/>
 </resources>
 <connections/>
</ui>
